### ISSUE TITLE
Alert tones for new incidents

### ISSUE THREAD

**Issue by patrickfish** [patrickfish/pulsepoint-dashboard#6]
add a tone for incidents that pop on the dashboard?  Maybe a tone for "incidents" and a different tone for control burns.  It would be awesome to hear something and look up.

**Comment by patrickfish** [comment]
Dependent on #2

**Comment by patrickfish** [comment]
Initial work on this is done, will gather feedback and open a new issue for enhancements.
